composer.json

```diff
@@ -19,8 +19,9 @@
         "psr-4": {"": "src/"}
     },
     "require": {
-        "phpstan/phpstan-shim": "0.11.*",
-        "phpstan/phpstan-doctrine": "0.11.*",
-        "ocramius/proxy-manager": "2.2.*"
+        "phpstan/phpstan": "0.12.*",
+        "phpstan/phpstan-doctrine": "0.12.*",
+        "ocramius/proxy-manager": "2.2.*",
```

Comment:
@x86demon do we need dependency on ocramius/proxy-manager?

Reply:
removed